### ISSUE TITLE
[VT-12] vit needs .deb

### ISSUE THREAD

**Issue by taskwarrior** [scottkosty/vit#34]
_David Patrick on 2013-08-10T16:03:31Z says:_ 

 Vit is an excellent enhancement for taskwarrior, but sadly, many people just won't use something if they have to compile it first. 

If vit could be installed with a simple "apt-get install vit" (or synaptic, or software-center) a lot more users would benefit.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-08-10T16:03:31Z
Modified: 2015-11-29T12:01:33Z
```

**Comment by taskwarrior** [comment]
_Florian Baumann on 2013-08-15T11:07:11Z says:_ 

 
There actually is something packaged.

http://ftp-master.debian.org/new/vit_1.1-1.html

**Comment by taskwarrior** [comment]
_Scott Kostyshak on 2013-08-16T01:05:06Z says:_ 

 Florian Baumann wrote:
> There actually is something packaged.
> 
> http://ftp-master.debian.org/new/vit_1.1-1.html

Good find Florian. It is very useful to know this. Thank you.

**Comment by taskwarrior** [comment]
_Florian Baumann on 2013-09-02T10:56:52Z says:_ 

 The vit debian packaged has been accepted and is now available in stable :) 

http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=717271

**Comment by taskwarrior** [comment]
_Rowan Thorpe on 2015-11-29T12:01:33Z says:_ 

 Is there a reason why this is being kept open as "unresolved" in the backlog? In Debian I discovered vit version 1.2-2 (and consequently Taskwarrior) via idle Aptitude browsing, and am using it daily (so the packaging is "stable"). Perhaps this is staying open because you wish to do upstream .deb packaging/maintenance rather than relying on the Debian Developers' version?

**Comment by fdela** [comment]
This bug should probably be closed: debian stable has a vit package available, albeit somewhat old 1.3-beta 1
